src/main/resources/META-INF/rewrite/was-to-liberty.yml

```diff
@@ -52,6 +52,11 @@ recipeList:
       performStaticCall: true
 ---
 type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.liberty.WebSphereUnavailableSSOCookieMethod
+recipeList:
+  - org.openrewrite.java.RevokeSSOCookiesToLogout
+---
+type: specs.openrewrite.org/v1beta/recipe
 name: org.openrewrite.java.liberty.WebSphereUnavailableSSOTokenMethod
```

Comment:
Lacks a `displayName` and `description`, which we display in the docs, so best to have those available.